flow360/component/simulation/meshing_param/volume_params.py

```diff
@@ -102,10 +103,11 @@ class AutomatedFarfield(Flow360BaseModel):
                     Note: "user-defined" are left out due to scarce usage and will not be implemented.
     """
 
+    type: Literal["AutomatedFarfield"] = pd.Field("AutomatedFarfield", frozen=True)
     name: Optional[str] = pd.Field(None)
     method: Literal["auto", "quasi-3d"] = pd.Field(default="auto", frozen=True)
     private_attribute_entity: GenericVolume = pd.Field(
-        GenericVolume(name="__farfield_zone_name_not_properly_set_yet"), frozen=True
+        GenericVolume(name="__farfield_zone_name_not_properly_set_yet"), frozen=True, exclude=True
```

Comment:
Why do we exclude here?

Reply:
what is the reason to dump entities to JSON? Are they intended to be used by some other process? I found all these additional properties confusing for webUI team because they are trying to set them but they seem to be redundant.